schema.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE item (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  genre_id INT,
+  author_id INT,
+  label_id INT,
+  publish_date DATE,
+  archived BOOLEAN,
+  FOREIGN KEY (genre_id) REFERENCES genres (id),
+);
+
+CREATE TABLE movie (
+  id  INT,
+  silent BOOLEAN,
+	date_published Date,
+  FOREIGN KEY(id) REFERENCES item(id)
+);
+
+CREATE TABLE source (
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  name VARCHAR(255),
+);
+
+CREATE TABLE game (
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  multiplayer BOOLEAN
+  last_played_at DATE
+  publish_date DATE
+  archived BOOLEAN
+);
+
+CREATE TABLE author (
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  first_name VARCHAR(255)
+  last_name VARCHAR(255)
+)
+
+CREATE TABLE music_album (
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  on_spotify BOOLEAN,
+  publish_date DATE,
+  archived BOOLEAN
+);
+
+CREATE TABLE genre (
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  name VARCHAR(200)
+);
```

Comment:
- [ ] Please know that you should set up foreign keys correctly for the 4 main tables, music_album, game, movie and book, according to the UML diagram they are all linked to genre, label, author and source, this means that they should have foreign keys for all of them, kindly refer back to the UML diagram and add what's missing:
![image](https://user-images.githubusercontent.com/20567503/198043192-dadf8296-8890-42f5-8150-012d9ad3e1a5.png)